tests/tfengine/integration_test.go

```diff
@@ -12,41 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Prerequisites:
+//  - The terraform binary must be available in $PATH.
+//  - The environment variable RUN_INTEGRATION_TEST must be set to "true".
+//  - The environment variables BILLING_ACCOUNT and FOLDER_ID must be set.
+//  - The calling user (e.g. authenticated local user or service account) must have
+//    `roles/resourcemanager.projectCreator` on the project and
```

Comment:
On which project?

Reply:
Ops, was a typo. should have said folder (for the folder id above)